src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -262,10 +262,34 @@ private void WriteSubClientInternalConstructor(ConstructorSignature signature)
             _writer.Line();
         }
 
+        private void DeclareMethodParameter(CodeWriter writer, ConvenienceMethod convenienceMethod)
+        {
+            foreach (var parameterChain in convenienceMethod.ProtocolToConvenienceParameters)
+            {
+                if (parameterChain.Input?.Kind == InputOperationParameterKind.Flatten)
+                {
+                    var typeName = parameterChain.Convenience?.Type.Name;
+                    var paraName = parameterChain.Convenience?.Name;
+                    writer.Append($"{typeName} {paraName:D} = ");
+                    writer.Append($"new {typeName}(");
```

Comment:
I believe we need some assertion to make sure these are not null.
And BTW we could write these like this to ensure the namespace:
```suggestion
                    var type = parameterChain.Convenience?.Type;
                    var paraName = parameterChain.Convenience?.Name;
                    writer.Append($"{type} {paraName:D} = ");
                    writer.Append($"new {type}(");
```